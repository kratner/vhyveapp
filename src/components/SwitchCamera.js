import React from "react";
import { MdCameraFront, MdCameraRear } from "react-icons/md";
import styled from "styled-components";

const SwitchCameraContainer = styled.div`
  color: #fff;
  cursor: pointer;
  z-index: 10;
  &:hover {
    color: #fdde6b;
  }
`;

const SwitchCamera = props => {
  return (
    <SwitchCameraContainer className="switchcamera">
      {//Check if message failed
      props.selfie ? (
        <MdCameraFront
          onClick={props.onClick}
          size={props.size}
          title="Switch to Rear Camera"
        />
      ) : (
        <MdCameraRear
          onClick={props.onClick}
          size={props.size}
          title="Switch to Front Camera"
        />
      )}
    </SwitchCameraContainer>
  );
};

export default SwitchCamera;
