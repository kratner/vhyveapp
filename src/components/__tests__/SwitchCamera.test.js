import React from "react";
import { shallow } from "enzyme";
import SwitchCamera from "../SwitchCamera";

describe("SwitchCamera", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<SwitchCamera />);
    expect(wrapper).toMatchSnapshot();
  });
});
