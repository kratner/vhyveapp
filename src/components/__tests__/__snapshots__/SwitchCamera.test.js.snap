// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwitchCamera renders correctly 1`] = `ShallowWrapper {}`;
